.editorconfig

```diff
@@ -99,17 +102,38 @@ dotnet_style_collection_initializer = true:suggestion
 dotnet_style_coalesce_expression = true:suggestion
 dotnet_style_null_propagation = true:suggestion
 dotnet_style_explicit_tuple_names = true:suggestion
+dotnet_style_readonly_field = true:suggestion
+dotnet_style_prefer_is_null_check_over_reference_equality_method = true:suggestion
+dotnet_style_prefer_inferred_tuple_names = true:suggestion
+dotnet_style_prefer_inferred_anonymous_type_member_names = true:suggestion
+dotnet_style_prefer_conditional_expression_over_return = true:refactoring
+dotnet_style_prefer_conditional_expression_over_assignment = true:refactoring
+dotnet_style_prefer_auto_properties = true:suggestion
+csharp_prefer_simple_default_expression = true:suggestion
+
+dotnet_code_quality_unused_parameters = non_public:suggestion
+dotnet_code_quality.ca1802.api_surface = private, internal
+dotnet_code_quality.CA2208.api_surface = public
```

Comment:
Please add comments about that it is.

Reply:
I will consider `dotnet_code_quality.ca1802` and `dotnet_code_quality.CA2208` seperately from this PR